src/mmw/apps/geoprocessing_api/tasks.py

```diff
@@ -47,9 +47,18 @@ def start_rwd_job(location, snapping, data_source):
     rwd_url = 'http://%s:%s/%s/%f/%f' % (RWD_HOST, RWD_PORT, end_point,
                                          lat, lng)
 
+    query_string = '?'
```

Comment:
Now that there's more than one param, consider using [`urllib.urlencode()`](https://docs.python.org/2/library/urllib.html#urllib.urlencode)

Reply:
Sounds good! Added in e6bc740